specs/client/request_spec.py

```diff
@@ -86,25 +86,34 @@
             expect(self.request.http_request.data).to(
                 equal(b'{\"foo\": \"bar\"}'))
 
-        with it('should return X-HTTP-Method-Override as GET in HTTP header for some endpoints'):
-            self.request = Request({
-                'host': self.host,
-                'verb': 'POST',
-                'path': self.path,
-                'params': self.params,
-                'bearer_token': self.bearer_token,
-                'client_version': self.client_version,
-                'language_version': self.lang_version,
-                'app_id': self.app_id,
-                'app_version': self.app_version,
-                'port': self.port,
-                'ssl': self.ssl
-            })
+        with it('should return X-HTTP-Method-Override in header'):
```

Comment:
cause the self.path has been declared before so in this case I will suggest you use a new name, here is an example code, but the best way is to make the list as a global variable to reduce the redundancy of the code
```
        with it('should return X-HTTP-Method-Override as GET '
                'in HTTP header for some endpoints'):
            list_httpoverride = [
                '/v2/shopping/flight-offers',
                '/v1/shopping/seatmaps',
                '/v1/shopping/availability/flight-availabilities',
                '/v2/shopping/flight-offers/prediction',
                '/v1/shopping/flight-offers/pricing?',
                '/v1/shopping/flight-offers/upselling'
            ]

            for path in list_httpoverride:
                self.request = Request({
                    'host': self.host,
                    'verb': 'POST',
                    'path': path,
                    'params': self.params,
                    'bearer_token': self.bearer_token,
                    'client_version': self.client_version,
                    'language_version': self.lang_version,
                    'app_id': self.app_id,
                    'app_version': self.app_version,
                    'port': self.port,
                    'ssl': self.ssl
                })

                expect(self.request.http_request).to(be_a(HTTPRequest))
                expect(self.request.headers['X-HTTP-Method-Override']).to(
                    equal('GET')
                )
```